packages/client/src/AccountsClient.js

```diff
@@ -264,6 +270,7 @@ export class AccountsClient {
   }
 
   async requestPasswordReset(email?: string): Promise<void> {
```

Comment:
Yeah I was doing that because meteor was taking a optional argument. But from the client the email should be mandatory I think, my mistake :)